x-pack/plugins/monitoring/public/directives/logstash/pipeline_viewer/index.js

```diff
@@ -30,8 +30,8 @@ uiModule.directive('monitoringLogstashPipelineViewer', ($injector) => {
 
       scope.$watch('pipeline', (updatedPipeline) => {
         pipelineState.update(updatedPipeline);
-        const configViewer = (
-          <ConfigViewer
+        const pipelineViewer = (
```

Comment:
I would recommend renaming this, or even better, it needn't be a variable, just put it into the render argument directly. too many bugs are caused by having 2 things only differentiated by casing.

This is just a recommendation

Reply:
Sure thing, I'm fine with that. I'll ping you with an update commit.